### DISCUSSION TITLE
SensESP v3.0.0-beta.1 Released

### DISCUSSION THREAD

**Discussion by mairas** [SignalK/SensESP#727] [Announcements]
Hi all,

I have now merged the `dev-3` development branch into `main` and released version 3.0.0-beta.1. This is a major new version. While documentation hasn't been updated yet, I have listed the major changes (there are many!) and written some migration information in [the repo README](https://github.com/SignalK/SensESP/). I will also make corresponding update releases in related repos such as project templates etc.

Please have a go at it and report any issues and annoyances!

**Comment by pessimist51** [comment]
I'm probably being stupid but here goes.  Using platformio I added the sensESP library to a new project.  This resulted in the following error -
---------------------
Resolving esp32dev dependencies...
Library Manager: Installing signalk/SensESP @ ^3.0.0-beta.1
UnknownPackageError: Could not find the package with 'signalk/SensESP @ ^3.0.0-beta.1' requirements for your system 'linux_x86_64'
-----------------------
I tried the same on Windows and received a very similar error with "windows_amd64" substituted for the system.  I'm probably doing something wrong (I'm not familiar with platformio) but I can't for the life of me see what's wrong.
